analytics/terraform/emr-eks-karpenter/emr-eks.tf

```diff
@@ -0,0 +1,39 @@
+
+module "emr_containers" {
+  source = "./modules/emr-eks-containers"
+
+  enable_emr_on_eks     = true
```

Comment:
Do we still need "enable_emr_on_eks" after separating it out?

Reply:
We can certainly take that out. I will take a look. This sub-module is going to be new repo.